src/server.rs

```diff
@@ -71,11 +74,22 @@ impl MetricsService {
             return None;
         }
 
+        fn pct(resp: &LastMetricsResult, p: f64) -> f64 {
+            // get the pth percentile and convert from msg/Msec to msg/sec
+            let b = resp.msg_rate.percentile(p).unwrap().start() as f64 / 1000000.;
+            let e = resp.msg_rate.percentile(p).unwrap().end() as f64 / 1000000.;
+            // return the bucket midpoint
+            b + (e - b) / 2.
```

Comment:
@ballard26 maybe look at this in particular, not sure if there was a specific reason for choosing the left, right, right bounds for min, max, median in the old code.